libraries/pom.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```

Comment:
Please do not alter spacing within or between existing lines. This makes the review take longer than it should, as I have to wade through what is an actual change vs what is only a whitespace change.

Reply:
it was wrong formatted previously. Someone commited wrong style so I want to fix that issue :)

Reply:
Understood. I've had difficulties with this file recently since the formatting was changed to the wrong style. Several PRs had merge conflicts in this file initially, as is the case here. You'll need to pull down the upstream changes and resolve the merge conflicts and push again :)

Reply:
done